test/AppNotificationBuilderTests/APITests.cpp

```diff
@@ -52,6 +52,22 @@ namespace Test::AppNotification::Builder
             VERIFY_ARE_EQUAL(builder.BuildNotification().Payload(), expected);
         }
 
+        TEST_METHOD(AppNotificationBuilderSetTimeStamp)
+        {
+            std::wstring dateTimeFormat{ L"%d/%m/%Y %H:%M:%S" };
+            std::wistringstream ss(L"22/12/2016 01:12:10");
+
+            struct tm dateTime {};
+            ss >> std::get_time(&dateTime, dateTimeFormat.c_str());
+            std::time_t time{ mktime(&dateTime) };
```

Comment:
Make a string to convert to struct...seems like a lot of work instead of just `struct tm dateTime{ x, y, z, ... };`

Also, do `std::get_time` etc deal in localtime and locale formatting? Results would depend on the machine and user context when run.

Setting the `tm` struct directly seems simpler and more reliable